rct229/rulesets/ashrae9012019/__init__.py

```diff
@@ -86,6 +86,7 @@
     "prm9012019rule66m62": "section6rule7",
     "prm9012019rule16x33": "section6rule8",
     "prm9012019rule22c86": "section6rule9",
+    "PRM9012019Rule86d29": "section6rule12",
```

Comment:
Could we set this rule id to lower cases? Does not seem it follows the convention.